x-pack/plugins/threat_intelligence/cypress/integration/sources/indicators.spec.ts

```diff
@@ -111,4 +114,18 @@ describe('Indicator page search', () => {
       cy.get(EMPTY_STATE).should('exist').and('contain.text', 'No results');
     });
   });
+
+  it('should have the default selected field, then update when user selects', () => {
```

Comment:
this test isn't the most useful... I wanted to verify that the barchart was updated on selection of a field in the field selector component, but I can't get a `data-test-subj` to work on the barchart component at all... plus I'm not exactly sure what to test exactly in the barchart... as it should be handled by the charts library